src/coreclr/jit/emitarm64.h

```diff
@@ -701,6 +715,24 @@ static bool isValidSimm4_MultipleOf32(ssize_t value)
     return (-256 <= value) && (value <= 224) && (value % 32 == 0);
 };
 
+// Returns true if 'value' is a legal signed multiple of 2 immediate 5 bit encoding (such as for LD1H).
```

Comment:
Nit: these comments should probably say "unsigned" instead of "signed".

Reply:
Yea it should say unsigned.